stories/banner.stories.js

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import { Story, Demo, Description, Code, comment } from './storybook/blocks';
+import './banner.scss';
+
+const title = 'Banner';
+
+export default { title };
+
+export const defaultBanner = () => (
+  <Story>
+    <Demo>
+      <div role="alert" className="Manifold__Banner">
+        <div className="Manifold__Banner__Icon">
+          <svg aria-label="info" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+            <g fill-rule="evenodd" clip-rule="evenodd">
+              <path d="M12 3a9 9 0 100 18 9 9 0 000-18zM1 12C1 5.925 5.925 1 12 1s11 4.925 11 11-4.925 11-11 11S1 18.075 1 12z" />
+              <path d="M12 11a1 1 0 011 1v4a1 1 0 11-2 0v-4a1 1 0 011-1zM11 8a1 1 0 011-1h.01a1 1 0 110 2H12a1 1 0 01-1-1z" />
+            </g>
+          </svg>
+        </div>
+        <p>Generic information uses a basic message bar.</p>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Default</h1>
+      <p>
+        The default banner surfaces info to the user without any urgency. Use it to give context
+        about an action.
+      </p>
+    </Description>
+    <Code
+      tabs={{
+        html: `<!-- banner -->
+<div role="alert" className="Manifold__Banner">
```

Comment:
💯! Love these a11y attributes. This is the best learning opportunity we can give to improve usability of our front-end.

Reply:
Looks like I just forgot to move that change into the stories. Is there a way you suggest keeping the stories' code tabs inline with the component? Maybe it's just manually copying.

Reply:
For me, manually copying is always the first step, and that’s fine until we’ve committed to something. But when copying becomes burdensome, you could create ad-hoc components like in the Typography story (you don’t have to look at it closely; just an example of creating another React component in this file that handles the repeating bits that you don’t `export`).

Reply:
But even then, sometimes if the stories are _just_ different enough (like Button) it’s easier to manage manual text. Depends!